pom.xml

```diff
@@ -83,6 +83,11 @@
         <sonar.issue.ignore.multicriteria.e1.resourceKey>oakpal-checks/pom.xml
         </sonar.issue.ignore.multicriteria.e1.resourceKey>
 
+        <!-- the minimum AEM 6.5 version being supported -->
+        <aem.classic.api.version>6.5.10.0002</aem.classic.api.version>
+        <!-- the minimum AEMaaCS version being supported -->
+        <aem.sdk.api.version>2023.6.12255.20230608T053118Z-230400</aem.sdk.api.version>
```

Comment:
not every customer is that up2date unfortunately. Do we really require such a new cloud version? In any case we should highlight this in the https://adobe-consulting-services.github.io/acs-aem-commons/pages/compatibility.html.

Reply:
Updated to the first one of this year (2023.1.10675.20230113T110236Z-220900)